.circleci/config.yml

```diff
@@ -0,0 +1,57 @@
+version: 2.1
+jobs:
+  build-node15:
+    docker:
+      - image: circleci/node:15-browsers
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run: yarn
+      - run: yarn run ci
+
+  build-node14:
+    docker:
+      - image: circleci/node:14-browsers
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run: yarn
+      - run: yarn run ci
+
+  build-node12:
+    docker:
+      - image: circleci/node:12-browsers
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run: yarn
+      - run: yarn run ci
+
+  deploy-npm:
+    docker:
+      - image: circleci/node:14-browsers
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run: yarn --pure-lockfile
+      - run: yarn run ci
+      - run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
+      # To help debug when lerna detects changes to the working tree and fails the publish
+      - run: git status
+      - run: yarn run release:publish
+
+workflows:
+  version: 2
+  voter:
+    jobs:
+      - build-node12

```

Comment:
You're sure Node 10 isn't needed? I'm still personally on that version for work stuff. 🤷‍♂️

Reply:
We can always add it later, but node10 will reach EOL in ~2 months, so maybe it is not worth it

Reply:
Sounds fair.